tests/rbd/rbd_utils.py

```diff
@@ -981,3 +999,40 @@ def verify_migration_abort(rbd, pool, image):
         f"Image {image} is not found in pool {pool}, abort image migration is successful"
     )
     return 0
+
+
+def verify_image_feature_exist(rbd, pool, image, feature_name):
+    """Verify image feature enable.
+    This method will verify for the image feature is exist on an image.
+
+    Args:
+        rbd: rbd object
+        pool: name of the pool
+        image: name of the image
+        feature_name: image feature which need to be verify
+
+    return :
+        int: The return value. 0 for success, else 1
```

Comment:
[Optional]
0 if enabled, 1 if disabled

this kind of doc would help better I think
We write doc to make it easier for anyone to understand/get gist of the code, if possible, please make changes.